tests/integration/operatorhub/cmd_service_test.go

```diff
@@ -292,4 +283,37 @@ spec:
 			helper.MatchAllInOutput(jsonOut, []string{msg, msgWithQuote})
 		})
 	})
+
+	Context("When deleting Operator backed service", func() {
+		It("should be able to delete a service existing on the cluster and fail otherwise", func() {
+			operators := helper.CmdShouldPass("odo", "catalog", "list", "services")
+			etcdOperator := regexp.MustCompile(`etcdoperator\.*[a-z][0-9]\.[0-9]\.[0-9]-clusterwide`).FindString(operators)
+			helper.CmdShouldPass("odo", "service", "create", etcdOperator, "--crd", "EtcdCluster")
+
+			// now verify if the pods for the operator have started
+			pods := helper.CmdShouldPass("oc", "get", "pods", "-n", project)
+			// Look for pod with example name because that's the name etcd will give to the pods.
+			etcdPod := regexp.MustCompile(`example-.[a-z0-9]*`).FindString(pods)
+
+			ocArgs := []string{"get", "pods", etcdPod, "-o", "template=\"{{.status.phase}}\"", "-n", project}
```

Comment:
The namespace actually picking random available active project. It will lead to fake for sure. Please define a teardown steps for example:

```
JustBeforeEach(func() {
        preSetup()
})
JustAfterEach(func() {
       cleanPreSetup()
})
```

Reply:
I totally forgot that. Thanks! Fixed.